### PR TITLE
add a few new website

### LINKED ISSUE #445: New site to add: [www.optifine.online]
**What site do you want to be added?**
https://www.optifine.online/

**What game does the site repost content for?** (Example: Minecraft, Subnautica, etc)
Minecraft

**Do you have anything else worth mentioning?**
also redistributes op auto clicker

### LINKED ISSUE #446: New site to add: [optifine.es]
**What site do you want to be added?**
optifine.es

**What game does the site repost content for?** (Example: Minecraft, Subnautica, etc)
Minecraft

**Do you have anything else worth mentioning?**
No

### LINKED ISSUE #458: New site to add: [mc-addons.com]
**What site do you want to be added?**
mc-addons.com

**What game does the site repost content for?** (Example: Minecraft, Subnautica, etc)
Minecraft PE

**Do you have anything else worth mentioning?**
no

### PR DESCRIPTION
**Pull request by Brandonbr1**
Closes #446 Closes #458 Closes #445

https://minecraft-pocket.com/ steals mod from mcpedl.com

### CODE REVIEW

File: lists/minecraft.yaml
Comment (dada513) [inline lists/minecraft.yaml:2064]: set notes to `Fake Optifine website`

**Review by dada513 — CHANGES_REQUESTED** [review]

**Comment by dada513** [comment]
add this to the PR description to properly link issues:

Closes #446 Closes #458 Closes #445

**Comment by Brandonbr1** [comment]
@dada513  did all of the changes😀😊 you asked me too

**Comment by dada513** [comment]
Thanks, I'll merge when I'm on my PC

Btw, don't close the issues manually, when linked in the PR they will auto close on merge 🙂

File: lists/minecraft.yaml
Comment (dada513) [inline lists/minecraft.yaml:2080]: remote the `-` at the end

**Review by dada513 — APPROVED** [review]
other than the last tiny review, seems good

**Comment by Brandonbr1** [comment]
@dada513 ok fixed everything you asked me in your review

**Comment by dada513** [comment]
ok waiting for berrysauce review

**Review by berrysauce — CHANGES_REQUESTED** [review]
Reason (reason: Illegal redistribution) is missing for minecraft-pocket.com and monster-mcpe.com

**Comment by Brandonbr1** [comment]
> Reason (reason: Illegal redistribution) is missing for minecraft-pocket.com and monster-mcpe.com

@berrysauce  changes done!

**Review by berrysauce — APPROVED** [review]
Thanks for making the changes!

**Pull request merged**
